tests/unit/test_dockerclient.py

```diff
@@ -200,6 +203,15 @@ def test_user(self):
         flags = Util.parse_additional_flags(argument_string)
         assert flags.user == "nobody"
 
+    def test_dns(self):
+        argument_string = r"--dns 1.2.3.4"
+        flags = Util.parse_additional_flags(argument_string)
+        assert flags.dns == ["1.2.3.4"]
+
+        argument_string = r"--dns 1.2.3.4 --dns 5.6.7.8"
+        flags = Util.parse_additional_flags(argument_string)
+        assert flags.dns == ["1.2.3.4", "5.6.7.8"]
```

Comment:
Please also add a test to ensure that if no dns is set, the flags.dns is really empty. We had some problems with other flags for that in the past.